spec/models/feedback_spec.rb

```diff
@@ -3,75 +3,27 @@
 RSpec.describe Feedback, type: :model  do
   subject(:feedback) { Fabricate.build(:feedback) }
 
-  let(:valid_feedback_token) { 'valid_feedback_token' }
-  let(:invalid_feedback_token) { 'invalid_feedback_token' }
-  let(:chapter) { Fabriate(:chapter) }
-
-  it { should respond_to(:request) }
-  it { should respond_to(:rating) }
-  it { should respond_to(:suggestions) }
-  it { should respond_to(:coach) }
-  it { should respond_to(:tutorial) }
-
   context 'validations' do
     context '#rating' do
-      it 'should not be blank' do
-        feedback = Fabricate.build(:feedback, rating: nil)
-
-        expect(feedback).to_not be_valid
-        expect(feedback).to have(1).error_on(:rating)
-      end
-
-      it 'should accept numbers from 1 to 5' do
-        (1..5).each do |rating|
-          feedback = Fabricate.build(:feedback, rating: rating)
-
-          expect(feedback).to be_valid
-        end
-      end
-
-      it 'should be invalid with number higher than 5' do
-        feedback = Fabricate.build(:feedback, rating: 6)
-
-        expect(feedback).to_not be_valid
-        expect(feedback).to have(1).error_on(:rating)
-      end
-
-      it 'should be invalid with number lower than 1' do
-        feedback = Fabricate.build(:feedback, rating: 0)
-
-        expect(feedback).to_not be_valid
-        expect(feedback).to have(1).error_on(:rating)
-      end
-
-      it 'should be numerical' do
-        feedback = Fabricate.build(:feedback, rating: 'alpha')
-
-        expect(feedback).to_not be_valid
-        expect(feedback).to have(1).error_on(:rating)
-      end
+      it { is_expected.to validate_presence_of(:rating) }
+      it { is_expected.to validate_inclusion_of(:rating).in_range(1..5).with_message(/can't be blank/) }
     end
 
-    context '#coach' do
-      it "accepts member with 'coach' role" do
+    it { is_expected.to validate_presence_of(:tutorial)}
+
+    context '#valid_member_has_coach_role?' do
+      it 'accepts coach members' do
         feedback = Fabricate.build(:feedback, coach: Fabricate(:coach))
 
         expect(feedback).to be_valid
       end
 
-      it "is invalid with member with 'student' role" do
-        feedback = Fabricate.build(:feedback, coach: Fabricate(:student))
-
-        expect(feedback).to_not be_valid
-        expect(feedback).to have(1).error_on(:coach)
-      end
-
-      it "should display correct error message when member with 'student' role given" do
+      it 'rejects none-coach members' do
         feedback = Fabricate.build(:feedback, coach: Fabricate(:student))
 
         feedback.valid?
 
-        expect(feedback.errors.messages[:coach]).to include("Coach member doesn't have 'coach' role.")
+        expect(feedback.errors[:coach]).to include("Coach member doesn't have 'coach' role.")
```

Comment:
This is wrong. A legacy from how things used to work. 
Anyo member can be a coach at a workshop and they do not be subscribed to any groups.

It's not related to your PR though so I can either merge or be happy to accept a change removing this validation entirely.

Reply:
I removed the validation completely....